redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/DatabaseServerV4Endpoint.java

```diff
@@ -134,7 +142,7 @@ DatabaseServerV4Response deleteByName(
     @ApiOperation(value = DatabaseServerOpDescription.DELETE_MULTIPLE_BY_CRN, notes = DatabaseServerNotes.DELETE_MULTIPLE_BY_CRN,
             nickname = "deleteMultipleDatabaseServersByCrn")
     DatabaseServerV4Responses deleteMultiple(
-        @ApiParam(DatabaseServerParamDescriptions.CRNS) Set<String> crns
+        @ValidCrn @ApiParam(DatabaseServerParamDescriptions.CRNS) Set<String> crns
```

Comment:
A question: does this validator also work for Set<String>?

Reply:
Thanks for bringing this up, I need to check this.

Reply:
Looks like if I move the annotation inside the Set it works, so:

```
Set<@ValidCrn String>
```

So that's what I've done.